valhalla/sif/edgelabel.h

```diff
@@ -399,7 +410,8 @@ class EdgeLabel {
   uint64_t not_thru_ : 1;
   uint64_t deadend_ : 1;
   uint64_t on_complex_rest_ : 1;
-  uint64_t spare_0 : 2;
+  uint64_t closure_pruning_ : 1;
```

Comment:
since we dont do this for the unidirectional algorithms, why not add this only to the BDEdgelabel like not_thru_pruning is. you can take some bits from the oppedgeid it doesnt need 63bits, the max id is only 21+22+3 bits wide.

Reply:
Yeah I started off by putting it in the `BDEdgelLabel` class. But since `DynamicCost::Allowed()` accepts an `EdgeLabel` (its also used in unidirectional astar variants which don't use `BDEdgeLabel`), I kept it in the base class.

Reply:
yeah i was thinking that since we dont care about it in unidirectional we didnt need it there but you're right since the allowed method has to take the base class, its easier to access it there. the other option is to just put the function for getting teh value from the label in the base class but actually store the boolean in the BDEdgeLabel.

then the base class always returns `true` but the BDEdgeLabel returns the actual value of the boolean/bit by overriding the getter function

Reply:
Added a `TODO` to move the `not_thru_pruning` flag to the base class.